tools/docs/patch-template.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+import jinja2
+
+from baselayer.app.env import load_env
+
+_, cfg = load_env()
+
+# open the openapi.json file
+# and patch the doc/api.html.template file with the new content
+with open('openapi.json') as f:
+    openapi_spec = json.load(f)
+
+# here, we could add a list of servers to the openapi_spec
+# for the scalar docs to let the users point to when testing an endpoint
+servers = cfg.get("docs.servers", [])
+# validate that servers have only 2 keys: url and description
+if not isinstance(servers, list) and servers is not None:
+    raise ValueError("API servers must be a list.")
+if servers is not None:
```

Comment:
Two checks that servers is not none; maybe handle that a different way on line 16

Reply:
done